Dockerfile

```
@@ -0,0 +1,13 @@
+FROM node:latest
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+
+RUN npm install
```

Para não ter a instalação de dependências necessárias, seria interessante setar para não baixar as dependências de desenvolvimento.
```suggestion
RUN npm install --production
```

---

Corrigido